### PR TITLE
Add psra sql copy fix

### PR DESCRIPTION
**Pull request by drotheram**
Major updates to the way PSRA and DSRA models are processed in PostGIS. The branch brings in several key changes to accommodate revisions to raw source data as well as major changes incorporating the meta \copy command for cloud deployment. This branch is a companion branch to the opendrr-api repository branch with the same name

### CODE REVIEW

**Comment by drotheram** [comment]
It would be best to merge this branch and the corresponding branch of opendrr-api simultaneously

**Review by wkhchow — APPROVED** [review]
major updates in psra_sql_copy_fix branch compared to master.  Looks good to me since the new branch was working before merge.

File: scripts/Create_all_tables_update.sql [hunk 1]
```diff
@@ -0,0 +1,277 @@
+-- update tables in one script
+
+-- create index on geometries tables
+CREATE INDEX IF NOT EXISTS geometry_aduid_pruid_idx ON boundaries."Geometry_ADAUID"("ADAUID");
```
Comment (jvanulde) [inline scripts/Create_all_tables_update.sql:4]: At some point, we may want to consider creating the indices after the data is loaded. Might not be practical in all cases though.

File: scripts/PSRA_runCreate_tables.py [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+        reader = csv.reader(f)
+        hmapColumns = next(reader)
+    hmapColumns = ','.join('"{0}"'.format(w) for w in hmapColumns)
+    hmapColumns = hmapColumns.replace('-','_')
+    hmapColumns = hmapColumns.replace('"lon"','lon')
+    hmapColumns = hmapColumns.replace('"lat"','lat')
+    # hmapColumns = hmapColumns.replace('"PGA_0.02"','PGA_0.02')
+    # hmapColumns = hmapColumns.replace('"PGA_0.1"','PGA_0.02')
+
+    with open("/usr/src/app/cHazard/{prov}/cH_{prov}_uhs.csv".format(**{'prov':args.province}), "r") as f:
```
Comment (jvanulde) [inline scripts/PSRA_runCreate_tables.py:34]: Can these paths be relative to the app directory? Something to try perhaps in the next version.

Comment (drotheram) [reply scripts/PSRA_runCreate_tables.py:34]: Yes, that wouldn't be too difficult

File: scripts/Create_physical_exposure_building_indicators_PhysicalExposure.sql [hunk 3]
```diff
@@ -64,24 +152,129 @@ b."CDUID" AS "cduid",
 b."CDNAME" AS "cdname",
 b."CSDUID" AS "csduid",
 b."CSDNAME" AS "csdname",
+b."CFSAUID" AS "fsauid",
+b."DAUIDt" AS "dauid",
+b."SACCODE" AS "saccode",
+b."SACTYPE" AS "sactype",
+CAST(CAST(ROUND(CAST(a.sauidlon AS NUMERIC),6) AS FLOAT) AS NUMERIC) AS "BldgLon",
```
Comment (jvanulde) [inline scripts/Create_physical_exposure_building_indicators_PhysicalExposure.sql:159]: We do a lot of CAST'ing in our scripts. Aren't the fields already NUMERIC?

**Review by jvanulde — APPROVED** [review]
Looks good to me. We may want to explore some optimizations in the next iteration of these scripts. Great work and it looks like we've streamlined a few things. 👍

**Pull request merged**